prow/cmd/validation-webhook/main_test.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"testing"
+)
+
+type SecretStore struct {
```

Comment:
Types defined in `*_test.go` files shouldn't typically be exported (capitalized).